common.sh

```diff
@@ -2,6 +2,13 @@
 
 eval "$(go env)"
 
+# Make dev-scripts work again
+export KNI_INSTALL_FROM_GIT=true
+export IRONIC_RHCOS_DOWNLOADER_LOCAL_IMAGE="https://github.com/openshift/ironic-rhcos-downloader"
```

Comment:
This landed in https://openshift-release.svc.ci.openshift.org/releasestream/4.3.0-0.ci/release/4.3.0-0.ci-2019-11-14-114835 so we can remove it from the workarounds now